tools/hermetic_grunt/Dockerfile

```diff
@@ -0,0 +1,54 @@
+# To build and run the hermetic grunt:
+# docker build -f tools/hermetic_grunt/Dockerfile -t uproxy-grunt . && \
+# docker run --rm -v $(pwd):/root/repository_root -it uproxy-grunt
+#
+# You have to run from the repository root. The output is written to the out/
+# directory.
+
+FROM ubuntu
```

Comment:
FYI, I ended up changing this to `selenium/node-chrome`, which comes with Chrome and makes some tests pass with an additional tweak to `entrypoint.sh`. I copied the trick from the existing `uproxy/Dockerfile`. There are still a few jasmine tests that are failing, but for some other reason.